src/Api/Feeds.php

```diff
@@ -97,4 +101,86 @@ public function getFeedDocument($feedDocumentId)
       'method' => 'GET',
     ]);
   }
+
+    /**
+     * @param $payload : Response from createFeedDocument Function. e.g.: response['payload']
+     * @param $contentType : Content type used during createFeedDocument function call.
+     * @param $feedContentFilePath : Path to file that contain data to be uploaded.
+     * @return string
+     * @throws \Exception
+     */
+  public function uploadFeedDocument($payload, $contentType, $feedContentFilePath)
```

Comment:
Well, the real problem with this code is that it needs to be outside the Api directory.
All the classes there are autogenerated and we can not guarantee this code will survive the next aerogeneration run.

COnsider forming new name space Spapi\Helper\ and place new class there called Feeder